utils/files_test.go

```diff
@@ -26,5 +26,6 @@ func TestIsDirEmpty(t *testing.T) {
 }
 
 func TestValidRawFilename(t *testing.T) {
-	assert.True(t, utils.ValidRawFilename("KJS80_INTERVJUFILM_BRØDRE_RCC_AUD_V02.wav"))
+	// Ø is not allowed
```

Comment:
Is it better if the method is called "Test_NorwegianLettersNotAllowed"?
I don't like comments and think they should rather be incorporated into the code. The main reason to write comments is to explain the Why, not the What.

Reply:
I.e. The comment is there to explain why that particular string is not allowed. There will probably be other strings later.